config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
```

Comment:
On Thu, Apr 28, 2016 at 01:53:53AM -0700, James Hunt wrote:
“This might not be as trivial as it sounds given that various distros
treat config file locations differently.”

I expect they also have different paths for their hypervisor binaries
;).  I'm not sure I have a clear picture of where the line is between
“belongs in config.json”, “belongs in /etc/{runtime}/… or similar”,
and “belongs in a runtime CLI option”.

Do we expect a single runtime on a single host to use different
hypervisor paths on a per-container basis?  That would seem to rule
out the /etc approach as a complete solution.

Do we expect bundle authors (vs. runtime callers) to care about which
hypervisor launches the container?  I'm not terribly familar with the
hypervisor ecosystem, but it sounds to me like HypervisorPersonality
[1] may be a better fit for config.json, and an /etc/{runtime}/…
setting would translate those personalities to paths that were
appropriate for the local system.

[1]: https://github.com/opencontainers/runtime-spec/pull/405/files/6194f0b24749bedaf3f384787b73cf1c9f899f4b#r61391724